pkg/app/app.go

```diff
@@ -39,6 +39,8 @@ type App interface {
 	Object() crv1alpha1.ObjectReference
 	// Uninstall deletes an app and all it's components from a Namespace.
 	Uninstall(context.Context) error
+	// Get ClusterLevelResources for the app
```

Comment:
```suggestion
	// GetClusterScopedResources returns the list of cluster scoped k8s resources created during app Install
```
header comment should start with the function name